requirements/admin.txt.in

```diff
@@ -0,0 +1,53 @@
+# The following list of immediate dependencies can be produced by install
+# snakefood and then running the following command line.
+#
+# Get a list of standard library packages so that we can omit those from our
+# requirements list.
+#
+# pip install stdlib-list
+#
+# python -c <<EOF  | cut --delimiter '.' --fields 1 | sort | uniq > std_packages
+#     from stdlib_list import stdlib_list
+#     print "\n".join(stdlib_list("2.7"))
+# EOF
+#
+# Install snakefood and use it to extract the non-dynamic Flocker imports.
```

Comment:
❓ Could this be automated with a small python script? Not crazy about having to copy a bash blob to VIM, remove the `#` signs, copy the bash blob again, paste it into a terminal, hope it works, etc.